pkg/auditlog/api/api.go

```diff
@@ -96,7 +96,8 @@ func (s *auditlogService) ListAuditLogs(
 	ctx context.Context,
 	req *proto.ListAuditLogsRequest,
 ) (*proto.ListAuditLogsResponse, error) {
-	_, err := s.checkRole(ctx, accountproto.Account_VIEWER, req.EnvironmentNamespace)
+	localizer := locale.NewLocalizer(locale.NewLocale(locale.JaJP))
```

Comment:
The next implementation will extract the locale information from the request header and set it. Is that correct?

Reply:
Not next, but I'll do it in the near future.

There are still a lot of parts that I have to remove. After removing them, I'll get the language data from headers.

Reply:
Ok, I was just confirming how the next implementation would be implemented.
Thanks.